ykman/otp.py

```diff
@@ -111,6 +111,68 @@ def messages(self):
         return [e.message() for e in self.errors]
 
 
+class SlotConfig(object):
+    def __init__(
+            self,
+            serial_api_visible=True,
+            allow_update=True,
+            append_cr=True,
+            pacing=None,
+            numeric_keypad=False,
+    ):
+        self.serial_api_visible = serial_api_visible
+        self.allow_update = allow_update
+        self.append_cr = append_cr
+        self.pacing = pacing
+        self.numeric_keypad = numeric_keypad
+
+
+class _SlotConfigContext(object):
+
+    def __init__(self, dev, cmd, conf):
+        st = ykpers.ykds_alloc()
+        self.cfg = ykpers.ykp_alloc()
+        try:
+            check(ykpers.yk_get_status(dev, st))
+            ykpers.ykp_configure_version(self.cfg, st)
+            ykpers.ykp_configure_command(self.cfg, cmd)
+        except YkpersError:
+            ykpers.ykp_free_config(self.cfg)
+            raise
+        finally:
+            ykpers.ykds_free(st)
+
+        if conf is None:
+            conf = SlotConfig()
+        self._apply(conf)
+
+    def _apply(self, config):
+        if config.serial_api_visible:
+            check(ykpers.ykp_set_extflag(self.cfg, 'SERIAL_API_VISIBLE'))
+        if config.allow_update:
+            check(ykpers.ykp_set_extflag(self.cfg, 'ALLOW_UPDATE'))
+        if config.append_cr:
+            check(ykpers.ykp_set_tktflag(self.cfg, 'APPEND_CR'))
+
+        # Output speed throttling
+        if config.pacing == 20:
+            check(ykpers.ykp_set_cfgflag(self.cfg, 'PACING_10MS'))
+        elif config.pacing == 40:
+            check(ykpers.ykp_set_cfgflag(self.cfg, 'PACING_20MS'))
+        elif config.pacing == 60:
+            check(ykpers.ykp_set_cfgflag(self.cfg, 'PACING_10MS'))
+            check(ykpers.ykp_set_cfgflag(self.cfg, 'PACING_20MS'))
+
+        if config.numeric_keypad:
+            check(ykpers.ykp_set_tktflag(self.cfg, 'USE_NUMERIC_KEYPAD'))
```

Comment:
I think this should be `ykp_set_extflag`

Reply:
Fixed...